nise/generators/ocp/ocp_generator.py

```diff
@@ -323,8 +367,42 @@ def _gen_pods(self, namespaces):
                         "mem_request_gig": mem_request_gig,
                         "mem_limit_gig": mem_limit_gig,
                         "pod_labels": self._gen_openshift_labels(),
+
+
+                    }
+
+                    ros_ocp_data_pods[pod] = {
+                        "namespace": namespace,
+                        "node": node.get("name"),
+                        "resource_id": node.get("resource_id"),
+                        "pod": pod,
+                        "deployment_name": self.fake.word() + "_" + self.fake.word(),
+                        "image_name": self.fake.word() + "-" + self.fake.word() ,
+                        "container_name":pod,
+                        "cpu_request_avg_container": randint(1,100),
```

Comment:
For all of these randints, the data that will be generated might be confusing. Might it make more sense to use something like `randint(25, 75)` for avg and sum, `randint(0, 25)` for min, and `randint(75, 100)` for max?